src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -656,6 +659,28 @@ else if (saturation > AVERAGE_SATURATION)
         }
     }
 
+    /**
+     * repair random equipped/held item with mending enchant.
+     *
+     * @param xp amount of xp available to mend with
+     * @return xp left after mending
+     */
+    private double applyMending(final double xp)
+    {
+        double localXp = xp;
```

Comment:
I.E. Is there any benefit to doing that, that I'm not seeing? =D (btw otherwise looks great)